odoo/models.py

```diff
@@ -6209,6 +6225,33 @@ def _recompute_field(self, field, ids=None):
             self.env.cache.invalidate([(field, records._ids)])
             self.env.remove_to_compute(field, records)
 
+    @api.model
+    def _traverse_field_path(self, path):
```

Comment:
I updated the method docstring, it was a leftover from a previous implementation and it didn't really describe the current state.

However I think `traverse` is the right term here, as the method iterates through the field in path.
There are similar methods in the codebase, doing simlar things, using the same term.

Reply:
My point is that your method has no side effects.  Its name should therefore describe _what it returns_ instead of _how it does it_.  My suggestion comes from the idea that the result is a sequence of field objects corresponding to the given sequence of field names.

For the record, two of the methods you pointed out are not public ones.  They are part of an implementation, and thus their names are less critical.  And the last one (`traverse_related`) iteratively applies a sequence of relational fields on the given record; it somehow _traverses_ those relations from the given record...

Reply:
I see. Point taken.
I've just pushed the changes ^^